src/kmeans.jl

```diff
@@ -2,13 +2,17 @@
 
 #### Interface
 
-mutable struct KmeansResult{T<:Real} <: ClusteringResult
-    centers::Matrix{T}         # cluster centers (d x k)
+# T is the eltype(centers)
```

Comment:
`T` no longer exists?

Reply:
Well we move around a lot. Before @alyst's comment it was `centers::AbstractMatrix{T} where T<:AbstractFloat` and now I guess the point was to move the `AbstractMatrix` out of the struct and into the parametrisation whence the `C<:AbstractMatrix{<:AbstractFloat}`.

I think at this point I'm a bit unclear what to do if different than that so would appreciate hints to specific alternatives if one such is desired :)

Reply:
ha or maybe you're just commenting about the comment itself... I'll fix that